BookingsApi/BookingsApi.DAL/Queries/GetAllocationHearingsBySearchQuery.cs

```diff
@@ -9,6 +9,7 @@ public class GetAllocationHearingsBySearchQuery : IQuery
         public Guid[] Cso { get;}
         public bool IsUnallocated { get; }
         public bool IncludeWorkHours { get; }
+        public bool ExcludeDurationsThatSpanMultipleDays { get; }
```

Comment:
If this operation is only used by the allocation job, can the query implement this change rather than making it an option in the query param?

Reply:
Yeah I try to avoid putting too many parameters in the constructors,  but the same query gets used in various places such as admin-web for building the forms, so wanted to leave the option available